### PR TITLE
improves RP

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
improves RP

# General Documentation
### Intent of your Pull Request
improves RP

### Why is this change good for the game?
improves RP

### Briefly describe your PR and the impacts of it, in layman's terms. 
improves RP

### What should players be aware of when it comes to the changes your PR is implementing?
They should notice the increase in RP.

### What general grouping does this PR fall under? 
improving RP

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
nooo, it's all improved RP after all

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
RP got improved by +1

# Changelog
:cl:  
rscadd: improved RP  
/:cl:

### CODE REVIEW

**Comment by Jarod1200** [comment]
no

**Comment by jcatjmeow** [comment]
Literal erp

**Comment by redmoogle** [comment]
What the fuck. **NO**

**Comment by Hopekz** [comment]
shitcode it doesn't even check for if you're looking at the groin area

**Comment by ChesterTheCheesy** [comment]
> 
> 
> shitcode it doesn't even check for if you're looking at the groin area

if i add that, i presume you'll speedmerge it?

**Comment by Hopekz** [comment]
> > shitcode it doesn't even check for if you're looking at the groin area
> 
> if i add that, i presume you'll speedmerge it?

Sure I'll keep clicking the merge button and hopefully, I get merge rights as I'm clicking it

**Comment by Marmio64** [comment]
what is wrong with you
